src/main/java/backend/Logic.java

```diff
@@ -231,7 +239,12 @@ public void removeUnusedModels(Set<String> reposInUse) {
                 .forEach(models::removeRepoModelById);
     }
 
-    public ImmutablePair<Integer, Long> updateRemainingRate(ImmutablePair<Integer, Long> rateLimits) {
+    /**
+     * Updates events related to remaining rate.
+     * @param rateLimits The api rate limits for updating of the remaining rate.
+     * @return Returns the rateLimits instance
```

Comment:
Need to explain the two values contained in the parameter. There is no other documentation of it.